src/docs/implementation/ArcCommanderManual/arc_get.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: "Clone an ARC from the DataHUB"
+published: 2022-12-20
+author: Martin Kuhl
+author_github: martin-kuhl
+add toc: false
+add support: false
+add sidebar: _sidebars/arcCommanderManualSidebar.md
+---
+
+Using `arc get -r` followed by a URL + ".git" enables you to clone an already existing ARC as a local repository, e.g.,
+
```

Comment:
Should we already mention the new lfs flag to prevent bad experience due to downloading very/too big shared ARCs? 

Also, for arc get, the authentification problem might occur or re-occur when the access TOKEN is expired as somebody set it like this. Therefore, the respective link would be also of interest here.

Reply:
Added as TODO to the markdown